base/test/test_base_image.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+
+import confluent.docker_utils as utils
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+IMAGE_DIR = os.environ.get("IMAGE_DIR") or os.path.join(CURRENT_DIR, "..")
+
+
+def get_dockerfile_path(image_dir):
+    return os.path.join(IMAGE_DIR, image_dir)
+
+
+class BaseImageTest(unittest.TestCase):
+
+    def setUp(self):
+        self.image = "{0}confluentinc/cp-base:{1}".format(os.environ["DOCKER_REGISTRY"], os.environ["DOCKER_TAG"])
+
+    def test_image_build(self):
+        self.assertTrue(utils.image_exists(self.image))
+
+    def test_java_install(self):
+        cmd = "java -version"
+        expected = 'OpenJDK Runtime Environment (Zulu 8.23.0.3-linux64) (build 1.8.0_144-b01)'
```

Comment:
Is this level of specificity guaranteed to continue passing? What happens if they update the base image with a new jdk build?

Reply:
Yeah, that's gonna break. I guess we could check for major version only, but on the other hand it's nice to know when this changes.

Reply:
yeah, elsewhere we just check the first two digits of the version